src/Core/Scheduler/Cache.js

```diff
@@ -134,18 +137,28 @@ class Cache {
      * @param {string} [key3]
      */
     delete(key1, key2, key3) {
-        const entry = this.data.get(key1);
-        if (entry == undefined) {
-            throw Error('Please specify at least a key of something to delete');
-        } else if (entry.get(key2) == undefined) {
+        const entry_1 = this.data.get(key1);
+        if (entry_1 !== undefined && !(entry_1 instanceof Map)) {
```

Comment:
You can ditch the `entry_1 !== undefined` because `undefined instanceof Map` results in `false`.

Reply:
There's logical `not` with `instanceof`, we need to check if `entry_1 `is defined.